api/api.go

```diff
@@ -1068,6 +1069,91 @@ func (api *api) Health(ctx context.Context) (*HealthResponse, error) {
 	return &HealthResponse{Alarms: alarms}, nil
 }
 
+func (api *api) GetNodeCommands() (*NodeCommandsResponse, error) {
```

Comment:
Minor: should we name it `GetCustomNodeCommands` (and related methods/types etc)?